### ISSUE TITLE
explorer: remove _ prefix from library functions identified using FLIRT

### ISSUE THREAD

**Issue by mike-hunhoff** [mandiant/capa#923]
prevents matching for library functions like `fopen` which IDA labels `_fopen`.

**Comment by williballenthin** [comment]
I'm not sure if this is an IDA-ism or not; though I agree in spirit with this change.

**Comment by mr-tz** [comment]
I think this is more a compile/link artifact? https://stackoverflow.com/questions/2627511/why-do-c-compilers-prepend-underscores-to-external-names

Should we maybe emit both symbols like we do for A/W APIs?

**Comment by williballenthin** [comment]
Agree, maybe we should do both, so you can use the common name and also wherever shows up in IDA verbatim.

**Comment by mike-hunhoff** [comment]
we also need to consider that some of these library functions already contain a `_` prefix: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/fopen-wfopen?view=msvc-170

**Comment by mr-tz** [comment]
I agree with Willi's comments in the PR, let's emit both.

For functions containing a `_` prefix in the documentation, these could show up with two `__` leading underscores, if I recall correctly.

**Comment by mike-hunhoff** [comment]
@mr-tz right and if we emit both our bases should be covered. I'll update the PR to support this in the viv and IDA extractors.
